case_studies/dc2/train_dc2.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# run the galaxy parameters and psf parameters extration notebook
+jupyter nbconvert --execute DC2_galaxy_psf_params.ipynb --to html
```

Comment:
I'm getting this error on this line: `ModuleNotFoundError: No module named 'GCRCatalogs'`

Would you run `poetry add --group dev lsstdesc-gcr-catalogs`?

Reply:
Same for `pyarrow`